lamindb/models/__init__.py

```diff
@@ -90,3 +91,7 @@
 ArtifactParamValue = ArtifactFeatureValue  # backward compat
 RunParamValue = RunFeatureValue  # backward compat
 Param = Feature  # backward compat
+
+from lamindb.models._django import patch_many_to_many_descriptor
+
+patch_many_to_many_descriptor()
```

Comment:
Last nit from me: Why do you re-export this into the `.models` namespace? Why not just call it within `_django` where you define it?

Alternatively, because we're subclassing `QueryManager` in `query_manager.py`, that would also a good place. `.add()` is an attribute of `QueryManager`.

Reply:
This was a deliberate choice as I thought that developers don't check `_django` but will see this patch in `models`. However, the function should not be in the `models` namespace. Therefore, I'll move it to `query_manager.py`. Thanks!